staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -331,9 +331,7 @@ type Initializer struct {
 
 const (
 	// TenantAll is the default argument to specify on a context when you want to list or filter resources across all tenants
-	TenantAll string = ""
-	// Sometimes we need to explicitly indicate the operation targets at all the tenants, not the empty string of the above TenantAll
-	TenantAllExplicit string = "all"
+	TenantAll string = "all"
```

Comment:
with this, "all" would be a reserved phrase and it should be checked against when regular user creates tenant

Reply:
Correct. It is already checked here:

![image](https://user-images.githubusercontent.com/51831990/108260770-92710900-7117-11eb-9630-afe1c5ff1b3a.png)